guides/micronaut-scope-types/java/src/test/java/example/micronaut/RefreshableScopeTest.java

```diff
@@ -0,0 +1,62 @@
+//tag::imports[]
+package example.micronaut;
+
+import io.micronaut.context.annotation.Property;
+import io.micronaut.core.type.Argument;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.BlockingHttpClient;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Property(name = "endpoints.refresh.enabled", value = StringUtils.TRUE) // <1>
+@Property(name = "endpoints.refresh.sensitive", value = StringUtils.FALSE) // <1>
+@MicronautTest // <2>
+class RefreshableScopeTest {
+    @Inject
+    @Client("/")
+    HttpClient httpClient; // <3>
+
+    @Test
+    void refreshableScopeIsACustomScopeThatAllowsABeansStateToBeRefreshedViaTheRefreshEndpoint() {
+//end::imports[]        
+/*
+//tag::testheader[] 
+       String path = "/";
```

Comment:
Extra space to align indention.

```suggestion
        String path = "/";
```